api/par-api.json

```diff
@@ -1146,14 +1218,14 @@
         ]
       }
     },
-    "/file-formats/{guid}": {
+    "/file-formats/{name}": {
       "delete": {
         "consumes": [],
-        "description": "Deletes the File Format specified by the GUID of the PAR Identifier",
+        "description": "Deletes the File Format specified by the Name part of the PAR Identifier",
         "parameters": [
           {
             "in": "path",
-            "name": "guid",
+            "name": "name",
```

Comment:
Path parameter names have changed from `guid` to `name` so even more reason to add another header var rather than use `guid` for a list of names.